src/cirrus/build.py

```diff
@@ -53,6 +53,18 @@ def build_parser(argslist):
         default=False,
         help='Use --upgrade to update the dependencies in the package requirements'
     )
+    parser.add_argument(
+        '--extra-requirements',
+        nargs="+", type=str,
+        dest='extras',
+        help='extra requirements files to install'
+    )
+    parser.add_argument(
+        '--no-setup-develop',
+        dest='nosetupdevelop',
```

Comment:
you mean the option name? --no_setup_develop ? I prefer to keep things consistent, stylewise around - or `_`